snapper/Hooks.cc

```diff
@@ -46,25 +73,33 @@ namespace snapper
 
 
     void
-    Hooks::create_snapshot(const string& subvolume, const Filesystem* filesystem)
+    Hooks::create_snapshot(const string& subvolume, const Filesystem* filesystem, const Snapshot& snapshot)
     {
 	grub(subvolume, filesystem, "--refresh");
+	run_scripts(std::list<string>({"create_snapshot", subvolume, std::to_string(snapshot.getNum())}));
```

Comment:
In general I do not like underscores here.

Reply:
so dash instead?

Reply:
Yes, dash is fine.